<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vtn_propiedades</class>
 <widget class="QDialog" name="vtn_propiedades">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>210</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_aceptar">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Aceptar</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="gb_propiedades">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>191</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Propiedades</string>
   </property>
   <widget class="QLabel" name="lb_altura">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Altura:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_ancho">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>51</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Ancho:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_tamano">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>51</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Tamaño:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_valormax">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Valor Máximo:</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbv_altura">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(ninguno)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbv_ancho">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(ninguno)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbv_tamano">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(ninguno)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbv_valormax">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>90</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>(ninguno)</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
